code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1032,12 +1032,13 @@
 	..()
 
 /datum/reagent/medicine/corazone
-	// Heart attack code will not do damage if corazone is present
-	// because it's SPACE MAGIC ASPIRIN
+	// Stops all organ damage effects from happening
+	// Also stops heart attacks
 	name = "Corazone"
 	id = "corazone"
 	description = "A medication used to treat pain, fever, and inflammation, along with heart attacks."
 	color = "#F5F5F5"
+	metabolization_rate = 0 // it gets filtered out manually in carbon code; rip
 
```

Comment:
Needs `process_flags = ORGANIC`.

You could add the organ healing stuff to liquid solder, I guess? In theory, does Corazone heal `ORGAN_ROBOTIC`?